protocol/parser/parser.go

```diff
@@ -155,14 +156,23 @@ func ParseRawBlock(rpcInput RPCInput, parsedInput *ParsedInput, defaultValue str
 	parsedInput.SetBlock(parsedBlock)
 }
 
-func parseInputWithLegacyBlockParser(rpcInput RPCInput, blockParser spectypes.BlockParser, source int) (string, error) {
+func parseInputWithLegacyBlockParser(rpcInput RPCInput, blockParser spectypes.BlockParser, source int, disableWarningLevelLog bool) (string, error) {
+	parseErrorLogLevel := utils.LAVA_LOG_WARN
+	if disableWarningLevelLog {
```

Comment:
an argument to a function about suppressing logs...

Reply:
Fixed